doc/tools.md

```diff
@@ -0,0 +1,51 @@
+# External tools
+
+## Setup
+
+In the top-level `index.js` of your ASAB UI application, load the Tools module 
+
+```
+const modules = [];
+
+...
+
+import ToolsModule from 'asab-webui/modules/tools';
+modules.push(ToolsModule);
+
+...
+
+ReactDOM.render((
+	<HashRouter>
+		<Application modules={modules} defaultpath="/" configdefaults={ConfigDefaults}/>
+	</HashRouter>
+), document.getElementById('app'));
+```
+
+
+## Setup in config file
+
+To load the configuration, external url's and paths of the image locations must be set.
+
+```
+{
+	"TOOLS":{
```

Comment:
"tools":  ...

And the content should be array (name is the attribute within).